### ISSUE TITLE
Unused constructor dependencies inspection should happen only in assignments

### ISSUE THREAD

**Issue by rentalhost** [kalessil/phpinspectionsea#1301]
| Subject        | Details                                                                       |
| :------------- | :---------------------------------------------------------------------------- |
| Plugin         | U2.0.15 |

Current behaviour
--
Not exactly a bug, but it will avoid a lot of inspections about the same problem by limiting current inspection to the assignment, instead of all usages.

![image](https://user-images.githubusercontent.com/421851/56488156-f131b100-64b3-11e9-95e3-2a1230789e10.png)

**Comment by funivan** [comment]
In this case error should not be rerported:
```php
class User{
 private $name='';
 function __construct(){
  file_put_contents($this->name . '.txt', 'data')
 }
}
```

This is not good. I think the main improvement will be highlighting the only first occurrence of the unused target property.

**Comment by kalessil** [comment]
Agree, only the first one should be reported.

**Comment by kalessil** [comment]
Fixed!
